src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1680,28 +1680,49 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		break;
 
 	case MAVLINK_MODE_EXTVISION:
-		configure_stream_local("ALTITUDE", 10.0f); // for avoidance
-		configure_stream_local("ATTITUDE", 25.0f);
-		configure_stream_local("ATTITUDE_TARGET", 10.0f);
-		configure_stream_local("CAMERA_TRIGGER", unlimited_rate); // for VIO
-		configure_stream_local("DISTANCE_SENSOR", 10.0f);		// for avoidance
+		configure_stream_local("HIGHRES_IMU", unlimited_rate);		// for VIO
+		configure_stream_local("TIMESYNC", 10.0f);
+
```

Comment:
Comment the fallthrough.

Reply:
I would like to have a consistent vision mode so even if we use the one with unlimited imu  then it still uses all the same streams as the minimal ext vision mode

Reply:
Yes that sounds good, I just mean add `// FALLTHROUGH` so someone doesn't come by later and add a break thinking you forgot it.